pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -301,11 +312,12 @@ func (vm *windows) transferFiles() error {
 		// of file that we want to transfer, WMCO team should cut a newer version of operator.
 		// The files are tightly coupled with the operator
 		// TODO: Remove this when we do in place upgrades
-		out, err := vm.Run("Test-Path "+dest+"\\"+filepath.Base(src), true)
+		path := dest + "\\" + filepath.Base(src)
+		fileExists, err := vm.FileExists(path)
 		if err != nil {
-			return errors.Wrapf(err, "error checking if file %s exists. output: %s", dest+"\\"+filepath.Base(src), out)
+			return errors.Wrapf(err, "error while checking if the file %s exists", path)
```

Comment:
I suggest removing the `while`.
Moreover the error is now going to get double wrapped with the same information. I suggest just returning the error here.